src/components/PaginatedTable/PaginatedTable.stories.mdx

```diff
@@ -0,0 +1,54 @@
+# PaginatedTable Component
+
+## Type: Simple
```

Comment:
@branquinhoaa this is a composite component. The reason for this classification is because it relies on two components: the Table and Pagination. Can you please make this correction? An example to reference is the documentation for the Dropdown component.